themes/phil-coffe-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+	<link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Italianno">
+	<link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Dosis">
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+	<script src="http://cdn.alloyui.com/3.0.1/aui/aui-min.js"></script>
```

Comment:
AUI should be readily avaliable in Liferay.  You should not need to include this additional script call for AUI.